scripts/install/opt/seagate/cortx/motr/libexec/m0reportbug-bundler

```diff
@@ -151,3 +165,79 @@ cd ..
 # The archives generated by `m0reportbug` are already compressed (.tar.gz).
 # compress them again to keep consistency with SB of other component.
 tar --remove-files -czf "${COMPONENT}_${bundle_id}.tar.gz" $HOSTNAME
+
+TEMP=( $( ls -ln "${COMPONENT}_${bundle_id}.tar.gz" ) )
+bundle_size=${TEMP[4]}
+
+if [[ $bundle_size -gt $sb_size_limit && $sb_size_limit -gt 0 ]]; then
```

Comment:
Add indentation for
 if [[ $bundle_size -gt $sb_size_limit && $sb_size_limit -gt 0 ]]; then 
 fi

Reply:
incorporated